sql-gremlin/src/test/java/org/twilmes/sql/gremlin/adapter/GremlinSqlIncompleteSelectTest.java

```diff
@@ -37,43 +39,58 @@ protected DataSet getDataSet() {
     @Test
     public void testProject() throws SQLException {
         runQueryTestResults("select name from person", columns("name"),
-                rows(r("Tom"), r("Patty"), r("Phil"), r("Susan"), r("Juanita"), r("Pavel")));
+                rows(r("Tom"), r("Patty"), r("Phil"), r("Susan"), r("Juanita"), r((Object) null)));
```

Comment:
Hm, I wonder if we should just not have them included in the result because the select is asking for name specifically so getting someone without a name would be unintuitive behavior.

Reply:
It is proper SQL to allow NULL values to come out